### ISSUE TITLE
Add summary stats for beds and ICU

### ISSUE THREAD

**Issue by hnykda** [epidemics/covid#288]
Followup for https://github.com/epidemics/covid/issues/192. Currently, we display total infections and active infections. This task is about extending this to:

* number of hospital beds needed
* number of deaths

See the https://github.com/epidemics/covid/issues/192 for proposals how to do it.

E.g:
> 20-50M active infections, which means 1.1M to 3M hospital beds and 500k-1M ICU units are needed.

> Overall there will be 200-800M infections, which can lead do 2-15M deaths.)

**Comment by janpipek** [comment]
Data should be exported here: https://github.com/epidemics/covid/issues/258

**Comment by hnykda** [comment]
from #258 we should have https://github.com/epidemics/epifor/pull/15/files#diff-6c3d86df2d4cc71a36d45075de3b7cb4R21712

**Comment by hnykda** [comment]
There is also this discussion: https://epidemicforecasting.slack.com/archives/CV1F9NB5K/p1585234929020700

**Comment by hnykda** [comment]
@mathijshenquet can we close this as it should be part of rates.txt added [here](Fixed in https://github.com/epidemics/epimodel/pull/25), right?

**Comment by mathijshenquet** [comment]
No we still need beds_per_100k, you can assign me this though.

**Comment by mathijshenquet** [comment]
Actually I didnt read this well and what was described in this Issue is now included.

**Comment by hnykda** [comment]
Yeah, this is actually a frontend part, so this should be closed until it's on frontend.

**Comment by hnykda** [comment]
AFAIK blocked now until  @jk3141 can give feedback
